content/docs/gke/gcp-e2e.md

```diff
@@ -651,93 +612,46 @@ The output from the training application includes the following:
   [TensorFlow Serving][tf-serving] component can read. Read on to see how
   to serve your model for prediction using TensorFlow Serving.
 
-### (Information) Running multiple jobs for the same model 
-
-In a production environment, it’s likely that you will need to run more than one
-training job for the model. Kubeflow gives you a simple deploy pipeline you can
-use to train new versions of your model repeatedly. When you have a new version
-to push:
-
-* Build a new image with a new version tag and a new path.
-* Run the `ks param set` command to modify the parameters to point to the new
-  version of your image:
-
-    ```
-    ks param set train name "train-"${VERSION_TAG}
-    ks param set train image ${TRAIN_IMG_PATH}
-    ```
-
-* Delete and re-apply the component to the cluster with the following commands:
-
-    ```
-    ks delete default -c train
-    ks apply default -c train
-    ```
-
-New model versions will appear in appropriately tagged directories in your
-Cloud Storage bucket.
-
 <a id="serve-model"></a>
 ## Serve the trained model
 
 Now you can put your trained model on a server and send it prediction requests.
-To put the model on a server, you can use two ksonnet components from the
-project files that you downloaded at the start of the tutorial. The components
-are in your `${WORKING_DIR}/components/my_ksonnet_app` directory. The
-relevant configuration files in that directory are
-`mnist-deploy-gcp.jsonnet` and `mnist-service.jsonnet`.
+Enter the `serving/GCS` directory.
+```
+cd $WORKING_DIR/serving/GCS
+```
 
-The `mnist-deploy-gcp` component uses the
-[`tf-serving` prototype][tf-serving-prototype] to spin up a prediction server. 
-The `tf-serving` prototype is the Kubeflow implementation of
-[TensorFlow Serving][tf-serving].
- 
-Unlike the training job that you ran earlier, the serving process does not
-require a custom image. The model files contain all the information that the 
-server needs.
+Set a different name for the tf-serving.
 
-Follow these instructions to point the serving component to the Cloud Storage
-bucket where your model is stored, so that the server can spin up to handle
-requests:
+```
+kustomize edit add configmap mnist-map-serving --from-literal=name=mnist-gcs-dist
+```
 
-1. Set a ksonnet parameter to define the path to your model on Cloud Storage:
+Set your model path
```

Comment:
Add a colon (:) at the end of sentences, rather than leaving them dangling. The colon helps the reader understand that the next code snippet does the thing you're asking them to do.

```
Set your model path:
```

Reply:
The above comment applies throughout the PR.